### ISSUE TITLE
MR2 telemetry: Recent bookmarks

### ISSUE THREAD

**Issue by rocketsroger** [mozilla-mobile/fenix#22103]
Add telemetry for the following: 
    Metric, boolean: Does the user see the Recent Bookmarks section on the homepage

**Comment by AndiAJ** [comment]
Hi @rocketsroger , I've just checked this matter on both Firefox 95.0.0 Beta 1 and  the latest Nightly build using a  using a OnePlus A3 (Android 6.0.1)

✅   Recent bookmarks section was shown to the user
```
        {
          "timestamp": 119552,
          "category": "recent_bookmarks",
          "name": "shown"
        }
```

On the other hand, the `recent_bookmarks_enabled` probe isn't generated in the metrics ping.

**Comment by rocketsroger** [comment]
> Hi @rocketsroger , I've just checked this matter on both Firefox 95.0.0 Beta 1 and the latest Nightly build using a using a OnePlus A3 (Android 6.0.1)
> 
> ✅ Recent bookmarks section was shown to the user
> 
> ```
>         {
>           "timestamp": 119552,
>           "category": "recent_bookmarks",
>           "name": "shown"
>         }
> ```
> 
> On the other hand, the `recent_bookmarks_enabled` probe isn't generated in the metrics ping.

To clarify, the problem here is `recent_bookmakrs_enabled`?  Thanks

**Comment by AndiAJ** [comment]
Exactly, the `recent_bookmakrs_enabled` with the boolean value isn't generated in the metrics ping.

**Comment by rocketsroger** [comment]
> Hi @rocketsroger , I think the probe is actually `customize_home.recently_saved` ?
> 
> If that's the case, verified as fixed on Firefox 95.0.0 Beta 1
> 
> ✅ Recent bookmarks enabled `"customize_home.recently_saved": true`
> 
> ✅ Recent bookmarks disabled `"customize_home.recently_saved": false`
> 
> [Logcat](https://github.com/mozilla-mobile/fenix/files/7472712/rcntsrcB.txt) [Glean dashboard](https://debug-ping-preview.firebaseapp.com/pings/AJ-0411001)

Yes, it should be covered by https://github.com/mozilla-mobile/fenix/issues/21239.  You're correct, it's `customize_home.recently_saved`.  Thanks
